src/components/format-number/format-number.test.ts

```diff
@@ -24,7 +24,7 @@ describe('<sl-format-number>', () => {
     ['de', 'de-CH', 'fr', 'es', 'he', 'ja', 'nl', 'pl', 'pt', 'ru'].forEach(lang => {
       it(`number has correct language format: ${lang}`, async () => {
         const el = await fixture<SlFormatNumber>(
-          html` <sl-format-number value="1000" lang="${lang}"></sl-format-number> `
+          html` <sl-format-number value="1000" lang="${lang}"></sl-format-date> `
```

Comment:
I don't think this was intentional. There are a few mismatched end tags. Possibly a bad merge, because I believe I fixed these in a prior commit.